packages/docs-reanimated/docs/fundamentals/getting-started.mdx

```diff
@@ -91,7 +91,32 @@ To learn more about the plugin head onto to [Reanimated babel plugin](/docs/fund
 
 </details>
 
-### Step 3: Clear Metro bundler cache (recommended)
+### Step 3: Wrap metro config with reanimated wrapper (recommended)
+
+Wrap your existing Metro configuration in the `metro.config.js` file with the `wrapWithReanimatedMetroConfig` function.
+
+```js
+const {
+  wrapWithReanimatedMetroConfig,
```

Comment:
is it still possible to change the name of this function?

i'd prefer `adaptReanimatedMetroConfig(config)` or something

as this arguably doesn't wrap the config tho, it changes the fields in the config object

Reply:
The name `wrapWithReanimatedMetroConfig` was coined to be on par with the visuals of using it:

```js
module.exports = wrapWithReanimatedMetroConfig(config); // config is "wrapped" here
```

I agree that `wrapWith` seems a bit odd. We could try `apply`, `add` or `mergeWith` instead.

Reply:
I wonder if `apply` or `merge` prefixes won't be even more confusing. They might suggest that some fields of the config would be overridden but the implementation of `wrapWithReanimatedMetroConfig` works in such a way that it takes into account users settings (including `symbolicator` field) and collapses these stack frames, which point to reanimated source code.

I previously used name `withReanimatedMetroConfig` (or `withReanimated`) but @tjzel suggested changing it to `wrapWithReanimatedMetroConfig`.

I have no strong opinion on how we should name this function so I can rename it if we decide on something.